test_programs/compile_success_no_bug/comptime_right_shift_no_overflow/src/main.nr

```diff
@@ -1,4 +1,4 @@
 fn main(x: Field) -> pub Field {
     let y = comptime { 1 >> 32 };
-    x + y
+    x + y as Field
```

Comment:
Can this be reverted?

Reply:
No - looks like we still remember `y` is a `u32` in this case.
Imo this is correct behavior anyway as the result of a right shift operation